components/localize/rtl-mixin.js

```diff
@@ -0,0 +1,14 @@
+export const RtlMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			_dir: { type: String, reflect: true, attribute: 'dir' }
+		};
+	}
+
+	constructor() {
+		super();
+		this._dir = document.documentElement.getAttribute('dir');
```

Comment:
Assuming an `undefined` value here doesn't result in `dir="undefined"` on the element?

Reply:
It looks ok, but doing a check seems a lot safer.